webapp/src/client/app/modules/web-client/rdp/web-client-rdp.component.ts

```diff
@@ -337,18 +369,37 @@ export class WebClientRdpComponent extends WebClientBaseComponent implements OnI
   }
 
   private callConnect(connectionParameters: IronRDPConnectionParameters): void {
-    this.remoteClient
-      .connect(
-        connectionParameters.username,
-        connectionParameters.password,
-        connectionParameters.host,
-        connectionParameters.gatewayAddress,
-        connectionParameters.domain,
-        connectionParameters.token,
-        connectionParameters.screenSize,
-        connectionParameters.preConnectionBlob,
-        connectionParameters.kdcProxyUrl,
-      )
+    const configBuilder = this.remoteClient
+      .configBuilder()
+      .withUsername(connectionParameters.username)
+      .withPassword(connectionParameters.password)
+      .withDestination(connectionParameters.host)
+      .withProxyAddress(connectionParameters.gatewayAddress)
+      .withAuthToken(connectionParameters.token);
+
+    if (connectionParameters.domain != null) {
+      configBuilder.withServerDomain(connectionParameters.domain);
+    }
+
+    if (connectionParameters.screenSize != null) {
+      configBuilder.withDesktopSize(connectionParameters.screenSize);
+    }
+
+    configBuilder.withExtension(displayControl(true));
+
+    if (connectionParameters.preConnectionBlob != null) {
+      configBuilder.withExtension(preConnectionBlob(connectionParameters.preConnectionBlob));
+    }
+
+    if (connectionParameters.kdcProxyUrl != null) {
+      configBuilder.withExtension(kdcProxyUrl(connectionParameters.kdcProxyUrl));
+    }
+
+    const config = configBuilder.build();
+
+    this.setKeyboardUnicodeMode(true);
```

Comment:
issue/question: The unicode mode is not supported by the RDP module (yet), and should not be default.

Reply:
I found the related code in IronRDP - https://github.com/Devolutions/IronRDP/blob/3029c8f9094094a3e0effa57a29d0df78c5e1823/crates/ironrdp-input/src/lib.rs#L307-L336, and I was able to trigger it in the webapp when the setting is enabled:
![image](https://github.com/user-attachments/assets/a45ae9e1-4613-43a6-8acf-2ebe48e9658b).

Is there anything more to it that I don't know?

Reply:
I found that we use here - https://github.com/Devolutions/IronRDP/blob/3029c8f9094094a3e0effa57a29d0df78c5e1823/web-client/iron-remote-desktop/src/services/remote-desktop.service.ts#L318-L325